### PR TITLE
Adding SessionID and IdentityID to  Debug end points

### PR DESCRIPTION
**Pull request by sojanpr**
Adding SessionId and IdentityId to  `v1/debug/connect` ,
`v1/debug/disconnect`  and `v1/debug/log`  end points in order to
support look up for   recent sessions.

@aaustin @dmitrig01

### CODE REVIEW

**Comment by dmitrig01** [comment]
:+1: but you should rpboably get someone who knows android better to review too

**Comment by Sarkar** [comment]
:+1:

File: Branch-SDK/src/io/branch/referral/BranchRemoteInterface.java [hunk 1]
```diff
@@ -86,7 +86,9 @@ public void connectToDebug() {
             post.put(Defines.Jsonkey.OSVersion.getKey(), sysObserver_.getOSVersion());
             post.put(Defines.Jsonkey.Model.getKey(), sysObserver_.getPhoneModel());
             post.put("is_simulator", sysObserver_.isSimulator());
-            callback_.finished(make_restful_post(post, prefHelper_.getAPIBaseUrl() + urlExtend, PrefHelper.REQ_TAG_DEBUG_CONNECT, prefHelper_.getTimeout(), false));
+            post.put(Defines.Jsonkey.SessionID.getKey(), prefHelper_.getSessionID());
```
Comment (aaustin) [inline Branch-SDK/src/io/branch/referral/BranchRemoteInterface.java]: Don't we need to worry about whether this has been set or not? What happens in poor connectivity cases?

Comment (aaustin) [inline Branch-SDK/src/io/branch/referral/BranchRemoteInterface.java]: same question as below

**Comment by aaustin** [comment]
:+1:

**Pull request closed without merging**